hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseTokenHandler.java

```diff
@@ -427,6 +427,36 @@ public static boolean isMetadataOnlyUpdateOp(@NonNull final TokenUpdateTransacti
         return op.equals(defaultWithMetadata);
     }
 
+    /**
+     * Check if TokenUpdateOp wants to update only some of the low priority keys or the metadata field
```

Comment:
Should metadata update be part of this?

Reply:
@thomas-swirlds-labs 
As I understood correctly the metadata HIP, we should allow the case where the `metadataKey` can update the `metadata` field by itself without requiring the admin key signature..? [Here](https://github.com/hashgraph/hedera-services/blob/12160-low-priority-to-an-invalid-key/hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateHandler.java#L459-L460) we allow for this when we enter the in this case `noOtherFieldThanLowPriorityKeyOrMetadataWillBeUpdated(op)`